Configure

```diff
@@ -509,7 +511,7 @@ my @disable_cascades = (
     "apps"              => [ "tests" ],
     "comp"		=> [ "zlib" ],
     sub { !$disabled{"unit-test"} } => [ "heartbeats" ],
-
+    sub { !$disabled{"saw"} } => [ "asm" ],
```

Comment:
I guess the intention is that if you configure with enable-saw that you get no-asm, but I don't think this does that?